samples/sampler/stories/qa/Scroller.js

```diff
@@ -1,12 +1,14 @@
 import {add, is} from '@enact/core/keymap';
 import Button from '@enact/sandstone/Button';
 import BodyText from '@enact/sandstone/BodyText';
+import {FixedPopupPanels, Panel} from '@enact/sandstone/FixedPopupPanels';
+import {Heading} from '@enact/sandstone/Heading';
 import ImageItem from '@enact/sandstone/ImageItem';
 import {InputField} from '@enact/sandstone/Input';
 import Item from '@enact/sandstone/Item';
-import Scroller from '@enact/sandstone/Scroller';
+import {Scroller, ContentContainerDecorator} from '@enact/sandstone/Scroller';
```

Comment:
Let's reorder this too just like you did for FixedPopupPanels. Thank you!

Reply:
It's fixed.